bsnes/target-libretro/program.cpp

```diff
@@ -506,6 +548,24 @@ auto Program::loadGameBoy(string location) -> bool {
 	return true;
 }
 
+auto Program::loadBSMemory(string location) -> bool {
+	string manifest;
+	vector<uint8_t> rom;
+	rom = loadFile(location);
+
+	if (rom.size() < 0x8000) return false;
+
+	auto heuristics = Heuristics::BSMemory(rom, location);
+	auto sha256 = Hash::SHA256(rom).digest();
+
+	bsMemory.manifest = manifest ? manifest : heuristics.manifest();
```

Comment:
Shouldn't this just be `bsMemory.manifest = heuristics.manifest();` ?

Reply:
Yes.